src/Data/Fin/Subset/Properties.agda

```diff
@@ -203,6 +256,171 @@ p⊆q⇒∣p∣<∣q∣ {p = outside ∷ p} {inside  ∷ q} p⊆q = ≤-step (p
 p⊆q⇒∣p∣<∣q∣ {p = inside  ∷ p} {outside ∷ q} p⊆q = contradiction (p⊆q here) λ()
 p⊆q⇒∣p∣<∣q∣ {p = inside  ∷ p} {inside  ∷ q} p⊆q = s≤s (p⊆q⇒∣p∣<∣q∣ (drop-∷-⊆ p⊆q))
 
+
+------------------------------------------------------------------------
+-- _⊂_
+
+p⊂q⇒p⊆q : ∀ {n} → {p q : Subset n} → p ⊂ q → p ⊆ q
+p⊂q⇒p⊆q = proj₁
+
+⊂-trans : ∀ {n} → Transitive (_⊂_ {n})
+⊂-trans (p⊆q , x , x∈q , x∉p) (q⊆r , _ , _ , _) = ⊆-trans p⊆q q⊆r , x , q⊆r x∈q , x∉p
+
+⊂-trans-⊆ : ∀ {n} → Trans (_⊂_ {n}) (_⊆_ {n}) (_⊂_ {n})
+⊂-trans-⊆ (p⊆q , x , x∈q , x∉p) q⊆r = ⊆-trans p⊆q q⊆r , x , q⊆r x∈q , x∉p
+
+⊆-trans-⊂ : ∀ {n} → Trans (_⊆_ {n}) (_⊂_ {n}) (_⊂_ {n})
+⊆-trans-⊂ p⊆q (q⊆r , x , x∈r , x∉q) = ⊆-trans p⊆q q⊆r , x , x∈r , λ x∈p → contradiction (p⊆q x∈p) x∉q
```

Comment:
The last term of this can be simplified to `x∉q ∘ p⊆q`

Reply:
Done